src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -338,6 +338,7 @@ const SelfExclusion = (() => {
             }
             return;
         }
+        getData();
```

Comment:
we already have the response of self exclusion in this function, can we pass it in instead of requesting it again?

Reply:
Alright, I changed the logic to reuse API response instead of making an additional call